report/upload_report.go

```diff
@@ -0,0 +1,151 @@
+package report
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"regexp"
+	"time"
+
+	"github.com/yuin/goldmark"
+	"github.com/yuin/goldmark/extension"
+	"github.com/yuin/goldmark/parser"
+	"github.com/yuin/goldmark/renderer"
+	"github.com/yuin/goldmark/renderer/html"
+)
+
+const (
+	YYYYMMDD = "2006-01-02"
```

Comment:
This is available as `time.DateOnly`: https://pkg.go.dev/time#pkg-constants

Reply:
Done, good to know